web/src/settings_users.js

```diff
@@ -437,9 +516,19 @@ function handle_edit_form($tbody) {
     });
 }
 
+function handle_filter_change($tbody, section) {
+    $tbody
+        .closest(".settings-section")
+        .find(".search")
+        .on("input.list_widget_filter", function () {
+            add_value_to_filters(section, "text_search", this.value.toLocaleLowerCase());
+        });
+}
```

Comment:
What does this block of code do? It looks like it pseudo-duplicates `set_up_event_handlers` in `ListWidget`...

Reply:
Ahh, OK, we're not using that `ListWidget` feature; added a comment before merging to address this.